### PR TITLE
Fixes | V1-263 - UI issues

### PR DESCRIPTION
**Pull request by ostapdans99**
- Frontend

1.  Added UI changes and features from this tasks list: https://trello.com/c/xUE1UD1E/271-v1-263-ui-issues

### CODE REVIEW

File: frontend/src/components/CoursesFilter/components/CustomSelect/styled.ts [hunk 1]
```diff
@@ -13,15 +12,16 @@ export const StyledSelect = styled(Select)({
...
     },
   },
   '& .MuiSelect-select': {
     padding: '8px 14px',
     backgroundColor: '#7676801F',
   },
   '& .MuiOutlinedInput-notchedOutline': {
-    border: '2px solid #CBBEBE',
+    border: '1px solid #CBBEBE',
+    borderWidth: '1px',
```
Comment (irina-sinitsyna) [inline frontend/src/components/CoursesFilter/components/CustomSelect/styled.ts:24]: I guess it's an extra line, as it's already included in the `border` property

**Review by irina-sinitsyna — APPROVED** [review]

File: frontend/src/components/CoursesFilter/components/CustomSelect/styled.ts (hunk 1 above)
Comment (ostapdans99) [reply frontend/src/components/CoursesFilter/components/CustomSelect/styled.ts:24]: agree

Comment (irina-sinitsyna) [reply frontend/src/components/CoursesFilter/components/CustomSelect/styled.ts]: 👍

**Review by irina-sinitsyna — DISMISSED** [review]

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -30,52 +31,55 @@ import {
...
+            element={
+              <AnonymousRoute>
+                <SignIn />
+              </AnonymousRoute>
+            }
+          />
+        </Routes>
+      </BrowserRouter>
+    </CoursesFiltersContextProvider>
+
```
Comment (PechurinaLena) [inline frontend/src/App.tsx:82]: empty line

Comment (ostapdans99) [reply frontend/src/App.tsx]: done

**Pull request merged**